GettingStarted.md

```diff
@@ -1,46 +1,65 @@
 
-## Install Postman App
+## Install Postman
 
-Instructions for installing the Postman app or using the web version can be found at https://www.postman.com/downloads/
+Instructions for installing the Postman app or using the web version can be found at [Installing and Updating](https://learning.postman.com/docs/getting-started/installation-and-updates/) in the Postman Learning Center.
 
-## Setting Up Enviornment
-Every root folder in [Platform API](api/), [Import](import/)
-, [Machine Learning](ml/) contains a template.json file. Import this file from the Environments tab.
+## Set up an environment via commercetools API client
 
-![image](https://user-images.githubusercontent.com/4946943/141699003-e989317e-41dc-42c9-b682-eb97f6c8fe6d.png)
+An environment is a set of variables that you can use in your Postman queries. It can be generated and downloaded from the Merchant Center using the API Client to match your project:
 
-This will create an empty variable set environment. 
-![image](https://user-images.githubusercontent.com/4946943/141699543-9f626cd3-5dcf-4b8d-94ad-f0045fc15b44.png)
+1. Create an API client following the instructions provided in [Create an API Client](https://docs.commercetools.com/merchant-center/api-clients#create-an-api-client) in the Merchant Center documentation.
+1. After clicking **Create API Client**, scroll down to the credential downloads section.
+1. Choose **Postman** from the dropdown menu.
+1. Click the **Download** icon.
 
+   ![image](/src/images/postman-environment-download.png)
+   This downloads a Postman environment file containing all relevant data for your project.
 
-Set the initial values for primary variables.
-* host : The host information can be [found here](https://docs.commercetools.com/api/general-concepts#hosts)
-* auth_url : The Authorization URL can be [found here](https://docs.commercetools.com/api/authorization#requesting-an-access-token-using-commercetools-oauth-20-server)
-* client_id : From your API Client. More information can be [found here](https://docs.commercetools.com/merchant-center/api-clients#create-an-api-client)
-* client_secret : From your API Client
-* project_key: your project key
+1. In Postman, click **File** > **Import** and upload the environment obtained from the Merchant Center.
 
+   The environment can now be chosen from the environment selector at the top-right of Postman.
+
+## Set up an environment manually
+
+Every root folder in [Platform API](api/), [Import](import/) and [Machine Learning](ml/) contains a `template.json` file.
+To set up an environment using these templates, perform the following steps:
+
+1. Import the `template.json` file, see [Importing data into Postman](https://learning.postman.com/docs/getting-started/importing-and-exporting-data/#importing-data-into-postman) in the Postman documentation for details.
+
+   This will create an empty variable set environment.
```

Comment:
I didn't do much with this section because I think it's obsolete. It is much easier to have the environment file generated by the API client and the template file is outdated anyway. 

I am waiting for a response from the team on this. If we still need it, I would definitely swap the image.